qiskit/compiler/assemble.py

```diff
@@ -32,7 +33,8 @@ def assemble(experiments,
              shots=1024, memory=False, max_credits=None, seed_simulator=None,
              qubit_lo_freq=None, meas_lo_freq=None,
              qubit_lo_range=None, meas_lo_range=None,
-             schedule_los=None, meas_level=2, meas_return='avg', meas_map=None,
+             schedule_los=None, meas_level=MeasLevel.CLASSIFIED,
+             meas_return=MeasReturnType.AVERAGE.value, meas_map=None,
```

Comment:
This should also accept an enum or string as input.

Reply:
Ok, I will make the changes. Thank you